include/external/block_timer.h

```diff
@@ -0,0 +1,30 @@
+//
+// Created by Alexander Taepper on 02.04.23.
+//
```

Comment:
I think Clion creates these automatically, but I think we don't need those, because we can look at git blame if we want to find this out. So I would suggest deleting this comment.